yugabyte/src/yugabyte/auto.clj

```diff
@@ -385,7 +387,8 @@
     (c/su (cu/stop-daemon! ce-master-bin ce-master-pidfile)))
 
   (stop-tserver! [db]
-    (c/su (cu/stop-daemon! ce-tserver-bin ce-tserver-pidfile)))
+    (c/su (cu/stop-daemon! ce-tserver-bin ce-tserver-pidfile))
+    (c/su (cu/grepkill! postgres-bin)))
```

Comment:
will it kill the process if it hangs?

Reply:
Yes, it uses `kill -9` as the default signal (can be overridden by providing signal code argument).